src/containers/search_index.jsx

```diff
@@ -91,7 +91,7 @@ class SearchIndex extends Component {
               </form>
             </Paper>
 
-            {!_.isEmpty(items) && <SearchResult />}
+            {!_.isEmpty(items) && <SearchResult searchInputValue={this.searchInput.input.value} />}
```

Comment:
It's not necessary to pass this value. The SearchResult component can get this value through this.props.query.term. Also, the way you used will make the component tightly coupled with the searchInput. It's not a good idea.

Reply:
Finished